components/Blog/styling/Blog.styles.ts

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+
+import {
+  LargeParagraph,
+  H2,
+  H3,
+  DefaultParagraph,
+} from 'components/atoms/typography.styles';
+
+export const TopDescription = styled(LargeParagraph)`
+  padding-bottom: 50px !important;
+  font-style: italic;
+
+  @media screen and (min-width: 768px) {
+    font-size: 36px;
+    line-height: 120%;
+    font-weight: 400;
+    padding-bottom: 55px !important;
+  }
+`;
+
+export const MainTitle = styled(H2)`
+  padding-bottom: 20px !important;
+
+  @media screen and (min-width: 768px) {
+    font-size: 48px;
+    line-height: 57.6px;
+    padding-bottom: 40px !important;
+  }
+`;
+
+export const MainContainer = styled.div`
+  padding: 30px 40px;
+  max-width: 860px;
+  margin: 0 auto;
+  
+  @media screen and (min-width: 768px) {
+    padding: 60px 40px 0;
+  }
+
+  ${H3},
+  ${LargeParagraph},
+  ${DefaultParagraph},
```

Comment:
Maybe this could be done with flex/grid with gap ? I don't like the idea of listing every possible component that can appear in the blog.

Reply:
I see your point, and this is actually the reason for the !important rules too. With this I wanted to avoid creating a custom styled component version of these elements in each of the subcomponents, only to give this same padding everywhere. Granted that when I considered this I did not have so many !important rules yet.

I don't see how flex/grid could help here because of the nested structure of the elements here.

Do you think it would be better to go with the redundancy of many 'StyledH3' and 'StyledLargeParagraph' elements with a bottom-padding of 30px?

Reply:
Yeah! This was a great idea!